internal/iptables/iptables.go

```diff
@@ -535,7 +535,7 @@ func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...st
 		if sets.NewString(fields...).IsSuperset(argset) {
 			return true, nil
 		}
-		klog.V(5).InfoS("DBG: fields is not a superset of args", "fields", fields, "arguments", args)
+		logrus.Debug("DBG: fields is not a superset of args", "fields", fields, "arguments", args)
```

Comment:
I think we can drop the prefix "DBG:" now as we are making this into a proper debug log level.